### ISSUE TITLE
demo

### ISSUE THREAD

**Issue by ArtskydJ** [ArtskydJ/tab-emitter#4]
This needs a sweet gh-pages demo!

**Comment by ArtskydJ** [comment]
``` js
var input = document.createElement('input')
document.body.appendChild(input)

input.addEventListener('keydown', function (ev) {
    setTimeout(function () {
        emitter.emit('color', input.value)
    }, 0)
})

emitter.on('color', function (color) {
    document.body.style.backgroundColor = color
})
```

**Comment by ArtskydJ** [comment]
Fixed in f496f0d6185accee4ed9338094de966c746bd421.
